storefront/package.json

```diff
@@ -37,6 +38,8 @@
     "react-syntax-highlighter": "^15.5.0",
     "rehype-highlight": "^6.0.0",
     "remark-frontmatter": "^4.0.1",
+    "remark-gfm": "^3.0.1",
+    "swr": "^2.1.3",
```

Comment:
I know a lot of teams in Digdir have adopted [React-query](https://dev.to/lvieira268/swr-vs-react-query-5el0) for datafetching. Should we use the same?